content/Interrupts.md

```diff
@@ -1,27 +1,27 @@
 ### IME - Interrupt Master Enable Flag (Write Only)
 
 ```
-0 - Disable all Interrupts
-1 - Enable all Interrupts that are enabled in IE Register (FFFF)
+0 - Disable all interrupts
+1 - Enable all interrupts that are enabled in the IE register (FFFF)
 ```
 
 The IME flag is used to disable all interrupts, overriding any enabled
-bits in the IE Register. It isn't possible to access the IME flag by
-using a I/O address, instead IME is manipulated by
-the following instructions/operations:
+bits in the IE register. It isn't possible to access the IME flag by
+using a I/O address. IME can be modified by
+the following instructions/operations only:
```

Comment:
I think "operations" should be changed here.

Reply:
Fixed